lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -2453,18 +2453,18 @@ instance LiftHandler ErrSubmitExternalTx where
             , errReasonPhrase = errReasonPhrase err400
             }
 
-instance LiftHandler ErrRemovePendingTx where
+instance LiftHandler ErrRemoveTx where
     handler = \case
-        ErrRemovePendingTxNoSuchWallet wid -> handler wid
-        ErrRemovePendingTxNoSuchTransaction tid ->
+        ErrRemoveTxNoSuchWallet wid -> handler wid
+        ErrRemoveTxNoSuchTransaction tid ->
             apiError err404 NoSuchTransaction $ mconcat
                 [ "I couldn't find a transaction with the given id: "
                 , toText tid
                 ]
-        ErrRemovePendingTxTransactionNoMorePending tid ->
+        ErrRemoveTxAlreadyInLedger tid ->
             apiError err403 TransactionNotPending $ mconcat
```

Comment:
Perhaps also worth to change into `TransactionNotInLedger`?

Reply:
Oops, yes fixed.